### ISSUE TITLE
Media Manager and android back button bugs

### ISSUE THREAD

**Issue by joshterrell805** [iFixit/iFixitAndroid#74]
## bugs

Media Manager:
- <del>If the user goes to Media Manager and uses the take a picture button, the 
  the picture gets added to images just fine. However, when the user clicks
  on the picture he just took, a ruler image appears instead of the image just
  taken. </del> 91ffcba
- <del>The picture taken from the above bullet is now rotated PI/2 radians clockwise.  </del> can't reproduce
- <del>If the user browses for a local picture and uploads it to the media manager, and
  then afterwards he clicks on that picture which was uploaded from within the
  Media Manager, the application crashes.</del> e83804d
- If the user uploads a png there is no notification that the filetype is 
  unsupported. Furthermore, if the png image added to images is clicked on,
  the app crashes. When the Media Manager refreshes after adding a png,
  the image is just gone.  --- I haven't been able to reproduce it, can you show me on your device?

Login/webview:
- When a logged in user views the answers of some device and clicks logout from
  within the webview, the user logs out from the webview but not the device.
  Furthermore, the user never logs back into the webview even if a different device
  is loaded, the user relogs from within the application, or the application is
  restarted.
## non-bugs / functionality

Media Manager:
- Cannot delete an image just added until after Media Manager has been refreshed.  -- This is expected, but kinda disarming.  should investigate further but not now
- Images in Media Manager are very pixily when downloaded from the server.  --- Should fix, but memory is still a problem.
- <del>If one adds an image to media manager and reloads the media manager before
  the image is ready on the server, the image appears as a ruler in the media
  manager. The image gets loaded on the media manager when it is ready, but it
  does not appear as a non-ruler until the app has been rotated or the media
  manager has been refreshed in some way. Maybe automatically refresh the
  image/media manager once the image is done loading.</del> Can't reproduce

Other:
- <del>When the menu is open and the back button is pressed, I expected it to just 
  close the menu and return to fullview of what I was looking at before. 
  However in adition to returning to fullview, the back button also causes
  the application to go back to the previous screen. If this is done on the
  first screen, the application exits.</del> 73e48fa
- If a teardown (and I'm guessing guides too) have many many steps, the scroll
  bubbles at the bottom of the screen will extend off the screen. Maybe create
  multiple rows of bubbles if the bubbles will extend off screen? See Samsung 
  Galaxy S4 Teardown.  --  Yea this has been an issue for a while, goint to convert to a titlepageindicator to match the guide edit interface
- Add a patrol message for when a normal user edits a guide/teardown.
